### PR TITLE
Rename component to event publisher proxy 

### PR DESCRIPTION
**Pull request by sayanh**
**Description**

Changes proposed in this pull request:

- Rename component to event publisher proxy from cloud event gateway proxy 

**Related issue(s)**

See https://github.com/kyma-project/kyma/issues/9381

### CODE REVIEW

**Review by nachtmaar — DISMISSED** [review]

File: components/event-publisher-proxy/README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
-# Cloud Event Gateway Proxy
+# Event Publisher Proxy
 
 ## Overview
 
-The Cloud Event Gateway Proxy receives Cloud Event publishing requests from the cluster workloads (microservice or Serverless functions) and redirects it to the Enterprise Messaging Service Cloud Event Gateway.
+The Event Publisher Proxy receives Cloud Event publishing requests from the cluster workloads (microservice or Serverless functions) and redirects it to an eventing platform. An example of an eventing platform would be SAP's One Event Bus.
```
Comment (k15r) [inline components/event-publisher-proxy/README.md:5]: do we want to mention OEB here?

File: components/event-publisher-proxy/config/300-deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+              valueFrom:
+                secretKeyRef:
+                  name: cloud-event-gateway-proxy
+                  key: client-secret
+            - name: TOKEN_ENDPOINT
+              valueFrom:
+                secretKeyRef:
+                  name: cloud-event-gateway-proxy
+                  key: token-endpoint
+            - name: EMS_PUBLISH_URL
```
Comment (k15r) [inline components/event-publisher-proxy/config/300-deployment.yaml:36]: I guess it confuses some people that we talk about OEB in the documentation, and use the acronym EMS here

File: components/event-publisher-proxy/README.md (hunk 1 above)
Comment (sayanh) [reply components/event-publisher-proxy/README.md]: No, I reverted the change back.

File: components/event-publisher-proxy/config/300-deployment.yaml (hunk 2 above)
Comment (sayanh) [reply components/event-publisher-proxy/config/300-deployment.yaml:36]: Sure, for now, it is EMS. Let's change it later when we decide on something else.

File: components/event-publisher-proxy/config/300-deployment.yaml [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+              valueFrom:
+                secretKeyRef:
+                  name: cloud-event-gateway-proxy
+                  key: ems-publish-url
+          image: github.com/kyma-project/kyma/components/cloud-event-gateway-proxy/cmd/cloud-event-gateway-proxy
```
Comment (marcobebway) [inline components/event-publisher-proxy/config/300-deployment.yaml:41]: This needs to change to the new name. In addition, the **cloud-event-gateway-proxy** occurrences need to change too.

The same applies for the **200-service.yaml** file.

**Review by marcobebway — CHANGES_REQUESTED** [review]

File: components/event-publisher-proxy/config/300-deployment.yaml (hunk 3 above)
Comment (sayanh) [reply components/event-publisher-proxy/config/300-deployment.yaml]: True, changed all the occurrences.

**Review by marcobebway — CHANGES_REQUESTED** [review]
Please delete the **200-deployment.yaml** file, as it is a duplicate of **300-deployment.yaml**.

**Review by marcobebway — CHANGES_REQUESTED** [review]
Please change the function doc, it is still has `Cloud Events Gateway Proxy`.

https://github.com/kyma-project/kyma/blob/0d332d87b4621f60f0a12ad3811d409ad040ab90/components/event-publisher-proxy/pkg/handler/handler.go#L52

**Comment by sayanh** [comment]
@marcobebway I fixed them

**Review by alexandra-simeonova — APPROVED** [review]

**Review by marcobebway — APPROVED** [review]

**Review by a-thaler — APPROVED** [review]

**Pull request merged**